vendor/knative.dev/pkg/configmap/testing/configmap.go

```diff
@@ -75,14 +75,23 @@ func ConfigMapsFromTestFile(t *testing.T, name string, allowed ...string) (*core
 	}
 	// With the length and membership checks, we know that the keyspace matches.
 
-	exampleBody := orig.Data[configmap.ExampleKey]
+	exampleBody, hasExampleBody := orig.Data[configmap.ExampleKey]
 	// Check that exampleBody does not have lines that end in a trailing space,
 	for i, line := range strings.Split(exampleBody, "\n") {
 		if strings.TrimRightFunc(line, unicode.IsSpace) != line {
 			t.Errorf("line %d of %q example contains trailing spaces", i, name)
 		}
 	}
 
+	// Check that the hashed exampleBody matches the assigned label, if present.
+	gotChecksum, hasExampleChecksumLabel := orig.Labels[configmap.ExampleChecksumLabel]
+	if hasExampleBody && hasExampleChecksumLabel {
+		wantChecksum := fmt.Sprint(configmap.Checksum(exampleBody))
```

Comment:
don't think it really matters, but this is probably our last chance to change it so I'll just mention out loud so we can be completely sure we want it this way: this is formatting the checksum as a uint rather than in hex, which would be (slightly) more concise and a bit (I think) more usual for a checksum. i.e. I think I'd've expected this to be `fmt.Sprintf("%08x", configmap.Checksum(exampleData))` (and that's what the [example_test of the crc32 package](https://golang.org/src/hash/crc32/example_test.go) does).

The difference is between e.g. `3000002650` and `b2d0685a` for the first config-map in the diff.